takeoffs-and-landings/src/main/java/io/spine/example/airport/tl/FlightAggregate.java

```diff
@@ -89,6 +89,7 @@ FlightRescheduled handle(RescheduleFlight command) {
         return withB(nothing());
     }
 
+    // #docfragment "TemperatureChanged"
     @React
     EitherOf2<FlightRescheduled, Nothing> on(@External TemperatureChanged event) {
```

Comment:
This method can be embedded using `start`/`end` patterns. Please avoid using named fragments when possible.

Reply:
I can't use `start/end` patterns here, because of several `@React` annotations.